Makefile

```diff
@@ -268,7 +274,6 @@ docker-build: test ## Build docker image with the manager.
 docker-push: ## Push docker image with the manager.
 	$(CONTAINER_TOOL) push ${IMG}
 
-GO_CONTAINER_CACHE = /tmp/dpu-operator-cache
```

Comment:
By removing this path, it breaks on my setup. Can we keep the temp file in /tmp?